### ISSUE TITLE
[sqllab]table schema select pattern is confusing

### ISSUE THREAD

**Issue by junlincc** [apache/incubator-superset#11741]
Currently, the pattern of selecting table schema in SQL Lab is very confusing.
1. it's a dropdown selection menu but the selected table is not populated in the box  
2. latest selected table goes all the way to the button in the panel, users have to scroll down to view column types, or take actions 
even with the data preview section under the editor, this flow is very unintuitive and hard to tell which table is actually being selected from the dropdown 

![flow](https://user-images.githubusercontent.com/67837651/99592507-2c080d80-29a5-11eb-8f98-8f34fd0dc191.gif)

**Comment by yousoph** [comment]
Agreed that it's confusing, thanks for bringing this up! Would like to spend some time soon on the left panel.
